authentication/jwtTokenProvider.go

```diff
@@ -10,9 +10,22 @@ var (
     defaultKeyFunc    jwt.Keyfunc = func(t *jwt.Token) (interface{}, error) { return jwtTestDefaultKey, nil }
 )
 
+type MyTimeClaims struct {
```

Comment:
Lets make the struct name different . Call it TimeClaims or TokenTimeClaims instead